(blog) Cake VSCode Extension 1.0.0 (#1319)
cake-build/website: input/blog/2020-12-10-cake-vscode-extension-release-1.0.0.md

---
title: Cake VSCode Extension Release 1.0.0

> category: News

Reviewer: I would use `Release Notes` category and use `News` just for not release related news (e.g. like joining DNF)

```suggestion
category: Release Notes
```

author: gep13
---

> The Cake Team are very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!

Reviewer: Shouldn't it be `The Cake Team is...`? 

```suggestion
The Cake Team is very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!
```

> With the upcoming release of Cake 1.0.0, and the transition in our documentation to recommend the usage of the .Net Tool Runner for Cake, the main focus of this version of the extension was to start using the .Net Tool Runner for all of the built in actions.  For example:

Reviewer: Use same naming as in docs and could be made a link

```suggestion
With the upcoming release of Cake 1.0.0, and the transition in our documentation to recommend the usage of the [Cake .NET Tool](/docs/running-builds/runners/dotnet-tool), the main focus of this version of the extension was to start using the Cake .NET Tool for all of the built in actions.  For example:
```

> * If you choose `Run Task` from the command palette, the chosen task will be executed using the .Net Tool Runner for Cake

Reviewer: Use same naming as in docs

```suggestion
* If you choose `Run Task` from the command palette, the chosen task will be executed using the Cake .NET Tool
```

> * If you use the code lens options to either `run task` or `debug task`, the .Net Tool Runner for Cake will be used

Reviewer: Use same naming as in docs

```suggestion
* If you use the code lens options to either `run task` or `debug task`, the Cake .NET Tool will be used
```

:::{.alert .alert-info}

> **NOTE:**

Reviewer: I would get rid of the **NOTE:** part here to be consistent with other annotation blocks

> If the .Net Global Runner for Cake isn't installed, it will be installed for you.

Reviewer: Use same naming as in docs

```suggestion
If the Cake .NET Tool isn't installed, it will be installed for you.
```

:::

> In all of these cases, you can override the default options to revert to using either the .Net Framework or the .Net Core Runner for Cake.

Reviewer: Use same naming as in docs and make them a link:

```suggestion
In all of these cases, you can override the default options to revert to using either the [Cake runner for .NET Framework](/docs/running-builds/runners/cake-runner-for-dotnet-framework) or the [Cake runner for .NET Core](/docs/running-builds/runners/cake-runner-for-dotnet-core).
```

### Release Notes

Check out the full set of [release notes](https://github.com/cake-build/cake-vscode/releases/tag/1.0.0) for this version.

### Release Video

> There is a [short video](https://youtu.be/7Ba-pAHzO9w) discussing the main aspects of this release.

Reviewer: Would it make sense to embed the video here instead of a link?

Author: I thought about it, but I wasn't sure if I should or not.  If you want me to, I am certainly happy to add this.

Reviewer: I think I would embed it since it saves one click and increases possibility that people are watching it

Author: Done :+1:
